### ISSUE TITLE
Video Feed Looping

### ISSUE THREAD

**Issue by danbayliss** [fwestenberg/reolink_dev#209]
**Describe the bug**
I've got six wired cameras and i've noticed sometimes the camera feed stops showing me a live video and shows me a 10 second video from the past which keeps looping. This seems to be the same time the camera motion sensor goes unavailable. The motion sensor seems to recover from the unavailable event and is accurate but the video feed continues showing the loop. I'm not sure why the cameras do this as they are wired and I've tested the cable. It seems to affect three of my six cameras, all at different times

**To Reproduce**
```
2021-07-05 08:30:24 ERROR (stream_worker) [homeassistant.components.stream.worker] Error demuxing stream: [Errno 1414092869] Immediate exit requested: 'rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78'
2021-07-05 08:30:36 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:30:56 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:31:26 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:32:06 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:32:56 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:33:57 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:34:47 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.35 error renewing the Reolink subscription
2021-07-05 08:35:07 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:36:27 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:37:57 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:39:37 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:41:27 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
2021-07-05 08:43:28 ERROR (stream_worker) [homeassistant.components.stream.worker] Error opening stream rtmp://192.168.0.35:1935/bcs/channel0_sub.bcs?channel=0&stream=0&token=02bbdc03e492e78
```

**Expected behavior**
Camera feed to recover and continue to show live view

**Screenshots**
![Screenshot_1](https://user-images.githubusercontent.com/54558493/124451344-3385f680-dd7d-11eb-8191-21e6c6ad3a5f.jpg)
Door & Side are live views, ignore the bottom right as that's when someone was last detected. The other three have all had issues overnight and are showing a short 10 second loop from when the camera become unavailable

**Environment:**
Please provide useful information about your environment, like:
 - Home Assistant version: core-2021.6.6
 - Reolink camera model: Two are RLC-410-5MP
 - Camera firmware number: v3.0.0.136_20121100
 - Reolink camera model: One is RLC-422
 - Camera firmware number: v3.0.0.136_20121105

**Comment by timota** [comment]
same happened with all of my cameras. 

Home Assistant version: 2021.7.1
Reolink camera model: RLC-410-5MP x 2, RLC-420-5MP
Camera firmware number: v3.0.0.65_20071000

**Comment by danbayliss** [comment]
I've now added my camera feeds as RTSP as well as the feeds from this integration. In the below screenshot the camera from the integration at the top is at 19:31 and the RTSP at the bottom continues to show me a live view
![Screenshot_3](https://user-images.githubusercontent.com/54558493/125849158-1b842823-b294-4ce1-b325-4509ca1ebad8.jpg)

One thing I have noticed is if I change 'Camera View' to 'Auto' it does show me a up to date view on the integration camera but it only updates every 10 seconds
![Screenshot_2](https://user-images.githubusercontent.com/54558493/125849373-f6982288-c175-4b8a-bf23-a59af2a8e52a.jpg)
![Screenshot_1](https://user-images.githubusercontent.com/54558493/125849379-9f417478-76ee-4114-97d6-ee61991764ca.jpg)

**Comment by danbayliss** [comment]
This has been fixed with 0.18
